RetroBar/Languages/Tiếng Việt.xaml

```diff
@@ -46,25 +51,35 @@
         <s:String>Luôn luôn</s:String>
         <s:String>Không bao giờ</s:String>
     </x:Array>
+    <s:String x:Key="middle_mouse_task_action">Khi _nhấn chuột giữa:</s:String>
+    <x:Array x:Key="middle_mouse_task_action_values" Type="s:String">
+        <s:String>Không làm gì</s:String>
+        <s:String>Mở một phiên bản mới</s:String>
+        <s:String>Đóng tác vụ</s:String>
+    </x:Array>
     <s:String x:Key="version">Phiên bản {0}</s:String>
     <s:String x:Key="visit_on_github">RetroBar trên GitHub</s:String>
     <s:String x:Key="taskbar_scale">Phóng lớn thanh tác vụ</s:String>
     <s:String x:Key="taskbar_scale_1x">100%</s:String>
     <s:String x:Key="taskbar_scale_2x">200%</s:String>
     <s:String x:Key="taskbar_scale_current">Cài đặt hiện tại: {0}%</s:String>
     <s:String x:Key="debug_logging">Kích hoạt ghi nhật ký gỡ lỗi</s:String>
+    <s:String x:Key="check_for_updates">Kiểm tra cập nhật</s:String>
     <s:String x:Key="ok_dialog">OK</s:String>
 
     <s:String x:Key="customize_notifications">Tùy chỉnh thông báo</s:String>
     <s:String x:Key="customize_notifications_info">RetroBar sẽ hiện các biểu tượng thông báo đang hoạt động hoặc quan trọng, đồng thời ẩn các biểu tượng không cần thiết. Bạn cũng có thể thay đổi hành vi của chúng trong danh sách dưới đây.</s:String>
     <s:String x:Key="customize_notifications_instruction">Chọn một mục nào đó, rồi chọn hành vi thông báo cho mục đó:</s:String>
     <s:String x:Key="hide_when_inactive">Ẩn nếu không hoạt động</s:String>
     <s:String x:Key="always_show">Luôn luôn hiện</s:String>
+    <s:String x:Key="always_hide">Luôn luôn ẩn</s:String>
+    <s:String x:Key="remove">Xoá</s:String>
     <s:String x:Key="name_heading">Tên</s:String>
     <s:String x:Key="behavior_heading">Hành vi</s:String>
+    <s:String x:Key="invert_heading">Đảo ngược</s:String>
 
     <s:String x:Key="start_text">Bắt đầu</s:String>
-    <s:String x:Key="start_text_xp">Bắt đầu</s:String>
```

Comment:
The Vietnamese version of Windows XP had the Start Menu button with the first letter capitalized.

Reply:
https://github.com/dremin/RetroBar/pull/324#discussion_r846705302

Reply:
Since we are trying to be more "faithful" to the `explorer.exe` strings from XP (and not Vista's) (after all, the string is used in the Windows XP theme, and if it conveys the correct meaning, it should be preserved), the first letter of the button text must be capitalized.

Refer to string ID 578 of `explorer.exe`'s `STRINGTABLE` resources.

Windows XP:
![xpbuttontextvi](https://github.com/user-attachments/assets/dec75d91-2dd1-4aa0-9eda-ec39c5bd007b)


Windows Vista:
![vistabuttontextvi](https://github.com/user-attachments/assets/7a0069c3-1b3f-46e3-bf6e-6c8070c253c2)